### PR TITLE
[FIX][DOC] _adjust_merged_values_orm: integer default case

### PR DESCRIPTION
**Pull request by mreficent**
Another case.

### CODE REVIEW

**Comment by mreficent** [comment]
@pedrobaeza

File: openupgradelib/openupgrade_merge_records.py [hunk 1]
```diff
@@ -242,18 +242,17 @@ def _adjust_merged_values_orm(env, model_name, record_ids, target_record_id,
...
-      * Text and Html fields:
-        - 'merge' (default): content is concatenated with an ' | ' as separator
-        - other value: content on target record is preserved
+      * Char, Text and Html fields:
+        - 'merge' (default for Text and Html): content is concatenated
+           with an ' | ' as separator
+        - other value (default for Char): content on target record is preserved
       * Integer, Float and Monetary fields:
-        - 'sum' (default): Sum all the values of the records.
+        - 'sum' (default for Float and Monetary): Sum all records's values.
```
Comment (pedrobaeza) [inline openupgradelib/openupgrade_merge_records.py]: Saxon genitive is for persons: https://escuelavictoria.es/saxon-genitive-2/, so you should use previous form.

File: openupgradelib/openupgrade_merge_records.py [hunk 2]
```diff
@@ -242,18 +242,17 @@ def _adjust_merged_values_orm(env, model_name, record_ids, target_record_id,
...
         - 'avg': Perform the arithmetic average of the values of the records.
         - 'max': Put the maximum of all the values.
         - 'min': Put the minimum of all the values.
-        - other value: content on target record is preserved
+        - other value (default for Integer): content on target record
+           is preserved
```
Comment (pedrobaeza) [inline openupgradelib/openupgrade_merge_records.py]: bad indentation

**Comment by mreficent** [comment]
Now it's full correct? 😏

**Comment by pedrobaeza** [comment]
I'm afraid not: when I said bad indentation, it was because that line had 3 spaces instead of 2, but you should preserve 2 spaces indentation on bullet lists, although docstring formatting is tricky, so maybe you can pass any kind of checker for being sure.

**Comment by pedrobaeza** [comment]
I have added #128 for a new interesting feature.

**Pull request merged**